notebooks/MetPy SkewT.ipynb

```diff
@@ -0,0 +1,442 @@
+    "skew = SkewT(fig, rotation=45)\n",
+    "skew.plot(p, T, 'r')\n",
+    "skew.plot(p, Td, 'g')\n",
+    "skew.plot_barbs(p[::3], u[::3], v[::3], y_clip_radius=0.03)\n",
+    "skew.ax.set_xlim(-30, 40)\n",
+    "skew.ax.set_ylim(1020, 100)\n",
+    "skew.plot_dry_adiabats(alpha=0.25, color='orangered')\n",
+    "skew.plot_moist_adiabats(alpha=0.25, color='tab:green')\n",
+    "skew.plot_mixing_lines(linestyle='dotted', color='tab:blue')\n",
+    "plt.title('{} Sounding'.format(station), loc='left')\n",
+    "plt.title('Valid Time: {}'.format(dt), loc='right')\n",
+    "#########################################################\n",
+    "#########################################################\n",
+    "\n",
+    "ax_hod = inset_axes(skew.ax, '30%', '30%', loc=1)\n",
+    "h = Hodograph(ax_hod, component_range=80)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we'll add a grid to the hodograph."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig = plt.figure(figsize=(9, 11))\n",
+    "skew = SkewT(fig, rotation=45)\n",
+    "skew.plot(p, T, 'r')\n",
+    "skew.plot(p, Td, 'g')\n",
+    "skew.plot_barbs(p[::3], u[::3], v[::3], y_clip_radius=0.03)\n",
+    "skew.ax.set_xlim(-30, 40)\n",
+    "skew.ax.set_ylim(1020, 100)\n",
+    "skew.plot_dry_adiabats(alpha=0.25, color='orangered')\n",
+    "skew.plot_moist_adiabats(alpha=0.25, color='tab:green')\n",
+    "skew.plot_mixing_lines(linestyle='dotted', color='tab:blue')\n",
+    "plt.title('{} Sounding'.format(station), loc='left')\n",
+    "plt.title('Valid Time: {}'.format(dt), loc='right')\n",
+    "#########################################################\n",
+    "#########################################################\n",
+    "\n",
+    "ax_hod = inset_axes(skew.ax, '30%', '30%', loc=1)\n",
+    "h = Hodograph(ax_hod, component_range=80)\n",
+    "\n",
+    "h.add_grid(increment=20)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Finally, we'll plot the wind data below 150 hPa on the hodograph and color it by speed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig = plt.figure(figsize=(9, 11))\n",
+    "skew = SkewT(fig, rotation=45)\n",
+    "skew.plot(p, T, 'r')\n",
+    "skew.plot(p, Td, 'g')\n",
+    "skew.plot_barbs(p[::3], u[::3], v[::3], y_clip_radius=0.03)\n",
+    "skew.ax.set_xlim(-30, 40)\n",
+    "skew.ax.set_ylim(1020, 100)\n",
+    "skew.plot_dry_adiabats(alpha=0.25, color='orangered')\n",
+    "skew.plot_moist_adiabats(alpha=0.25, color='tab:green')\n",
+    "skew.plot_mixing_lines(linestyle='dotted', color='tab:blue')\n",
+    "plt.title('{} Sounding'.format(station), loc='left')\n",
+    "plt.title('Valid Time: {}'.format(dt), loc='right')\n",
+    "#########################################################\n",
```

Comment:
Don't duplicate code, remove all lines from adding the grid and above, just leave the remaining two lines that plot the graph.

Reply:
So I tried that earlier and it appears that you need to have all of the code for the skewT/hodograph in the same jupyter notebook cell for it to work properly. I'm not sure if there's a way to disperse the plot generation across multiple cells like that.

Reply:
Definitely let me know if I'm missing something and there is a way to do this!

Reply:
Hmm Okay, let me see if I can get it to work...

Reply:
Okay, several things. 

1. I didn't notice until just now, you have created two notebooks called "MetPy SkewT.ipynb" -- one in the `/notebooks/` directory, and one in the `/notebooks/visualization` directory.  I think the one in the `/notebooks/` directory has most of the recent changes, so I'd suggest modifying that notebook, and then moving it into the `/notebooks/visualization/` directory on the next commit.
2. I went back and reviewed Section 2, and again you were duplicating a lot of code.  You just wanted the figure to redraw again, which you can do with the line `fig` at the end of your cell block. 
3. I've attached screenshots for both Sections 2 and 3 on what the code blocks should look like.  Please note them carefully as some of the changes I've made are very subtle.
4. Please implement the changes on the notebook, and then make sure there is only one copy of the notebook, located in the `/notebooks/visualization/` directory.

Thanks!
![Screen Shot 2021-01-09 at 1 46 05 PM](https://user-images.githubusercontent.com/21114460/104108407-9f315000-5281-11eb-939b-ec56ffea34aa.png)
![Screen Shot 2021-01-09 at 1 46 13 PM](https://user-images.githubusercontent.com/21114460/104108408-a0627d00-5281-11eb-969d-85d5d4eb4a42.png)

Reply:
Sounds good! Looks like I got a bit mixed up with the directory structure lol.

Reply:
No worries!

Reply:
I think I've incorporated all of those changes now--let me know if I missed any!

Reply:
Good!  Looks great.  Thanks for all your work.